build.sbt

```diff
@@ -129,8 +129,8 @@ lazy val `pekko-connectors` = project
     crossScalaVersions := List() // workaround for https://github.com/sbt/sbt/issues/3465
   )
 
-addCommandAlias("applyCodeStyle", ";scalafmtAll; scalafmtSbt; javafmtAll")
-addCommandAlias("checkCodeStyle", ";scalafmtCheckAll; scalafmtSbtCheck; javafmtCheckAll")
+addCommandAlias("applyCodeStyle", ";scalafmtAll; scalafmtSbt; javafmtAll; +headerCheckAll")
+addCommandAlias("checkCodeStyle", "+headerCreateAll; ;scalafmtCheckAll; scalafmtSbtCheck; javafmtCheckAll")
```

Comment:
`verifyCodeStyle` same as pekko

Reply:
I guess we should use a common name, I prefer checkCodeStyle. If you agree should I just make a PR in pekko to change it?

Reply:
@mdedetrich I'm Ok with both name,but better to keep consistence.

Reply:
Agreed, considering that pekko is the odd one out are you fine with me changing it there?

Reply:
@mdedetrich maybe that wil confuse some one contribute to both project?I mean akka & pekko, I saw @dongxuwang did the same too.